src/mocks/1.json

```diff
@@ -0,0 +1,2872 @@
+{
```

Comment:
I don't really like carrying mock data in the repo when it can be generated, but perhaps 128kb is not that much for the gain it provides. We'd have to stay on top of it so that it's valid.

Reply:
Dates, too

Reply:
good point. i'm not sure how i can also only embed it in MOCK mode.

that was just an example, it's not supposed to be generated **but we would use real libcore accounts potentially**.
we could just remove operations and set `[]` and libcore would re-fill it. so data could be minimal.